README.md

```diff
@@ -32,6 +32,25 @@ external evaluation engine in a vendor agnostic way.
 
 The OpenFeature project will include client libraries for common technology stacks including, but not limited to:
 
-* Golang
-* Java
-* JavaScript/TypeScript
+- Golang
+- Java
+- JavaScript/TypeScript (Node.js)
```

Comment:
It certainly is, and we don't want to forget about it. We just want to focus on the backend for now, since there's more considerations to be made for client side flags in general.